mscore/resourceManager.cpp

```diff
@@ -45,6 +45,59 @@ ResourceManager::ResourceManager(QWidget *parent) :
       MuseScore::restoreGeometry(this);
       }
 
+//---------------------------------------------------------
+//   ExtensionFileSize
+//---------------------------------------------------------
+
+ExtensionFileSize::ExtensionFileSize(const int i)
+      : QTableWidgetItem(stringutils::convertFileSizeToHumanReadable(i))
+      , _int(i)
+      {}
+
+bool ExtensionFileSize::operator<(const QTableWidgetItem& nextItem) const
+      {
+      QString currentText = text();
+      QString nextText = nextItem.text();
+
+      QChar currentSizeType = currentText[currentText.size() - 2];
+      QChar nextSizeType = nextText[currentText.size() - 2];
+
+      // the integer before unit
+      int currentInt = currentText.split(QRegularExpression("\\s+"))[0].toInt();
+      int nextInt = nextText.split(QRegularExpression("\\s+"))[0].toInt();
+
+      int currentSize, nextSize = 0;
+      
+      if      (currentSizeType == QChar(' '))   currentSize = currentInt                 ;
+      else if (currentSizeType == QChar('K'))   currentSize = currentInt * 0x400         ;
+      else if (currentSizeType == QChar('M'))   currentSize = currentInt * 0x100000      ;
+      else if (currentSizeType == QChar('G'))   currentSize = currentInt * 0x40000000    ;
+      else if (currentSizeType == QChar('T'))   currentSize = currentInt * 0x10000000000 ;
+
+      if      (nextSizeType    == QChar(' '))   nextSize    = nextInt                    ;
+      else if (nextSizeType    == QChar('K'))   nextSize    = nextInt    * 0x400         ;
+      else if (nextSizeType    == QChar('M'))   nextSize    = nextInt    * 0x100000      ;
+      else if (nextSizeType    == QChar('G'))   nextSize    = nextInt    * 0x40000000    ;
+      else if (nextSizeType    == QChar('T'))   nextSize    = nextInt    * 0x10000000000 ;
```

Comment:
Could you please move this conversion code to a separate function to avoid duplicating it?